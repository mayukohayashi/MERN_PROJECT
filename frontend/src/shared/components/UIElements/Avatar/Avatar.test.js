/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Avatar from './Avatar';

it('should render Avatar component', () => {
  expect(shallow(<Avatar />)).toMatchSnapshot();
});
