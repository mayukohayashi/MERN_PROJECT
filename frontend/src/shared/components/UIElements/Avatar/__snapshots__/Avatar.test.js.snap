// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Avatar component 1`] = `ShallowWrapper {}`;
